frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -237,17 +332,26 @@ function Application(): JSX.Element {
 						View Traces
 					</Button>
 					<Card>
-						<GraphTitle>Rate (ops/s)</GraphTitle>
-						<GraphContainer>
-							<FullView
-								name="operations_per_sec"
-								fullViewOptions={false}
-								onClickHandler={handleGraphClick('Rate')}
-								widget={operationPerSecWidget}
-								yAxisUnit="ops"
-								onDragSelect={onDragSelect}
-							/>
-						</GraphContainer>
+						{topLevelOperationsIsError && (
+							<Typography>
+								{(topLevelOperationsError as AxiosError).response?.data}
```

Comment:
there is a method axios.isAxiosError

Reply:
axios.isAxiosError is throwing Error `Cannot read the property of undefined`, better to keep it this way.

Thought?

Reply:
this is about enforcing about types

Reply:
Okay